src/app/src/mainwindow.cpp

```diff
@@ -229,6 +230,29 @@ void MainWindow::Initialize() {
 }
 
 
+/*
+* Runs when a new ni device is selected, re configure ni device leds
+*/
+void MainWindow::Resetup_ni_device(){
```

Comment:
this should take a `std::string` so the signal from advancedsetup can pass in the new device name